rosdep/python.yaml

```diff
@@ -5797,6 +5797,9 @@ python3-flake8-black:
     pip:
       packages: [flake8-black]
 python3-flake8-blind-except:
+  debian:
+    '*': [python3-flake8-blind-except]
+    bullseye: null
```

Comment:
If this is a platform which is known to not have the dependency available via apt and that we aren't building binaries on that platform we could use a fallback pip based key here. So that if someone tries to use it from source on that platform it will still work, and then it can be a hard dependency.

Ditto for below